<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quicklaunchConfig</class>
 <widget class="QWidget" name="quicklaunchConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure Quicklaunch</string>
  </property>
  <layout class="QGridLayout">
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="iconNamesVisibleLabel">
     <property name="text">
      <string>Show icon names:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>iconNamesVisibleCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="iconNamesVisibleCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="dialogEnabledLabel">
     <property name="text">
      <string>Enable popup:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>dialogEnabledCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="dialogEnabledCheckBox">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="maxRowsOrColumnsLabel">
     <property name="text">
      <string>Maximum rows:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>maxRowsOrColumnsSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="KIntSpinBox" name="maxRowsOrColumnsSpinBox">
     <property name="specialValueText">
      <string>unlimited</string>
     </property>
     <property name="suffix">
      <string notr="true"/>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="forceMaxRowsOrColumnsLabel">
     <property name="text">
      <string>Force row settings:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>forceMaxRowsOrColumnsCheckBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="forceMaxRowsOrColumnsCheckBox">
     <property name="text">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KIntSpinBox</class>
   <extends>QSpinBox</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>maxRowsOrColumnsSpinBox</tabstop>
  <tabstop>forceMaxRowsOrColumnsCheckBox</tabstop>
  <tabstop>iconNamesVisibleCheckBox</tabstop>
  <tabstop>dialogEnabledCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
